<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>74</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="SerialBox5" name="serial_widget" native="true"/>
      </item>
      <item>
       <widget class="SerialBox5_fix_baudrate" name="serial_widget_fix_baudrate" native="true"/>
      </item>
      <item>
       <widget class="SerialBox5_fix_baudrate_win7" name="serial_widget_fix_baudrate_win7" native="true"/>
      </item>
      <item>
       <widget class="SerialBox5_lite" name="serial_widget_lite" native="true"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SerialBox5_fix_baudrate</class>
   <extends>QWidget</extends>
   <header>serialbox5_fix_baudrate.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SerialBox5</class>
   <extends>QWidget</extends>
   <header>serialbox5.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SerialBox5_fix_baudrate_win7</class>
   <extends>QWidget</extends>
   <header>serialbox5_fix_baudrate_win7.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SerialBox5_lite</class>
   <extends>QWidget</extends>
   <header>serialbox5_lite.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
